File: Midas/MIDAS.GBX/GBBusinessObjects/PatientAccidentInfo/AccidentTreatment.cs
using Newtonsoft.Json;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MIDAS.GBX.BusinessObjects
{
    public class AccidentTreatment : GbObject
    {
        [JsonProperty("patientAccidentInfoId")]
        public int PatientAccidentInfoId { get; set; }

        [JsonProperty("medicalFacilityName")]
        public string MedicalFacilityName { get; set; }

        [JsonProperty("doctorName")]
        public string DoctorName { get; set; }

        [JsonProperty("contactNumber")]
        public string ContactNumber { get; set; }

        [JsonProperty("address")]
        public string Address { get; set; }
    }
}

File: patient-portal/src/app/patient-manager/cases/models/accident-treatment.ts
import { Record } from 'immutable';
import * as moment from 'moment';

const AccidentTreatmentRecord = Record({
    id: 0,
    patientAccidentInfoId: 0,
    medicalFacilityName: '',
    doctorName: '',
    contactNumber: '',
    address: '',
    isDeleted: false
});

export class AccidentTreatment extends AccidentTreatmentRecord {

    id: number;
    patientAccidentInfoId: number;
    medicalFacilityName: string;
    doctorName: string;
    contactNumber: string;
    address: string;
    isDeleted: boolean;

    constructor(props) {
        super(props);
    }

}

